config/samples/k8s_v1_chiadatalayer.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: k8s.chia.net/v1
+kind: ChiaDataLayer
+metadata:
+  labels:
+    app.kubernetes.io/name: chia-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: chiadatalayer-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
Should make a better sample here even though the samples aren't really advertised anywhere.